README.md

```diff
@@ -48,6 +50,43 @@ You can assign value to this variable using variables from inventory file. See e
     UAMS_OVERRIDE_HOSTNAME: "DEV_{{ override_hostname }}"
 ```
 
+
+### Managed locally Agents
+Optional environment variable `UAMS_MANAGED_LOCALLY` is used to set Agent as managed locally by configuration file. 
+Is designed to allow configuration of the UAMS Agent locally, without necessity of adding integrations manually from SWO page.
+
+If the UAMS Agent gets installed as a **managed locally** agent then it will wait for the local configuration file to be accessible. The default local configuration locations are:
```

Comment:
```suggestion
If the UAMS Agent gets installed as a **locally managed** agent then it will wait for the local configuration file to be accessible. The default local configuration locations are:
```